kiosk/kiosk_browser/connection.py

```diff
@@ -6,20 +10,23 @@
 
 check_connection_url = 'http://captive.dividat.com/'
 sleep_time_not_connected = 5
-sleep_time_connected = 60
+sleep_time_connected = 6
+sleep_time_proxy = 6
```

Comment:
These seem quite low to me. It's nice from a user's perspective, but sysadmins might find our device needlessly chatty when idle. Did you choose these durations with a specific scenario in mind?

Reply:
I don’t remember, I may have put small numbers for testing purpose, forgetting to put it back to 60. Putting it back to 60 in idle cases in https://github.com/dividat/playos/pull/54/commits/082b207e80f2cf871decfdcf2576021e1b15e5fd and improving the mechanic, logging a bit.